app/templates/view_buyers.html

```diff
@@ -135,6 +141,16 @@
       <input type="submit" value="Search" class="button-save">
   </form>
   </div>
+  <br />
+  <button id="edit-json-button" data-location="{{ url_for( '.brief_edit_json', brief_id=brief_id) }}">JSON Editor</button>
+  <script>
+      let editJsonButton = document.getElementById('edit-json-button');
+      editJsonButton.addEventListener('click', function(e) {
+          let location = e.target.getAttribute("data-location");
+          window.location = location;
```

Comment:
If you wanted to avoid inline JS code to change the window location, I think you could wrap the button in a `<form method="get" action="{{ url_for( '.brief_edit_json', brief_id=brief_id) }}">` to get the same redirect happening.